File: src/main/java/no/nav/pensjon/innsyn/domain/desc/DagpengerDescriptor.java
package no.nav.pensjon.innsyn.domain.desc;

import no.nav.pensjon.innsyn.entity.EntityDescriptor;

public class DagpengerDescriptor implements EntityDescriptor {

    private static final String ENTITY_NAME = "Dagpenger";

    private static final String[] PROPERTY_NAMES = {
            "Ferietillegg",
            "Barnetillegg",
            "Dagpenger",
            "Uavkortet grunnlag",
            "År dagpenger utbetalt",
            "Kilde",
            "Status",
            "Type",
            "Rapporttype"
    };

    public String getEntityName() {
        return ENTITY_NAME;
    }

    public String[] getPropertyNames() {
        return PROPERTY_NAMES;
    }
}

File: src/main/kotlin/no/nav/pensjon/innsyn/domain/desc/DagpengerDescriptor.kt
package no.nav.pensjon.innsyn.domain.desc

import no.nav.pensjon.innsyn.domain.Dagpenger
import no.nav.pensjon.innsyn.entity.EntityDescriptor

object DagpengerDescriptor : EntityDescriptor<Dagpenger>("Dagpenger",
        arrayOf(
                "Ferietillegg",
                "Barnetillegg",
                "Dagpenger",
                "Uavkortet grunnlag",
                "År dagpenger utbetalt",
                "Kilde",
                "Status",
                "Type",
                "Rapporttype"
        )
)